my_list.rb

```diff
@@ -0,0 +1,19 @@
+require_relative 'my_enumerable'
+class MyList
+  include MyEnumerable
+  def initialize(*list)
+    @list = list
+  end
+
+  def each(&block)
+    @list.each(&block)
+  end
```

Comment:
- [ ] [OPTIONAL] You should handle a case where a user forgot to pass a block, you can add a statement like the following example:
```ruby
    return to_enum(:each) unless block_given?
```
- [ ]  [OPTIONAL]  You are not supposed to use the `each` method from ruby, you should implement your own each method programmatically, this project's requirements say this `MyList class implements #each method that yields all members of @list in succession.`, could you try to respect this requirement please?

Reply:
We will attend to it, thank you